<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>biomedisa_segmentationb</class>
 <widget class="qMRMLWidget" name="biomedisa_segmentationb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>601</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="saCollapsibleButton">
     <property name="text">
      <string>Segment Anything</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="11" column="0">
       <widget class="QLabel" name="label_segmentAnythingActive">
        <property name="text">
         <string>Active: </string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QCheckBox" name="segmentAnythingActiveQCheckBox">
        <property name="toolTip">
         <string>Activates mouse interaction and automated segmentation.</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>segmentAnythingActive</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_pth">
        <property name="text">
         <string>model checkpoint: </string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="filePathLineEdit">
        <property name="SlicerParameterName" stdset="0">
         <string>modelCheckpointFile</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_pth">
        <property name="text">
         <string>model type: </string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="modelTypeLineEdit">
        <property name="SlicerParameterName" stdset="0">
         <string>modelType</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QPushButton" name="trainPredictorButton">
        <property name="toolTip">
         <string>Train the predictor.</string>
        </property>
        <property name="text">
         <string>Train predictor</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="biomedisaCollapsibleButton">
     <property name="text">
      <string>Biomedisa</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="101" column="0">
       <widget class="QLabel" name="label_allaxis">
        <property name="text">
         <string>allaxis: </string>
        </property>
       </widget>
      </item>
      <item row="101" column="1">
       <widget class="QCheckBox" name="allaxisQCheckBox">
        <property name="toolTip">
         <string>If pre-segmentation is not exlusively in xy-plane.</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>allaxis</string>
        </property>
       </widget>
      </item>
      <item row="102" column="0">
       <widget class="QLabel" name="label_sorw">
        <property name="text">
         <string>sorw: </string>
        </property>
       </widget>
      </item>
      <item row="102" column="1">
       <widget class="QSpinBox" name="sorwQSpinBox">
        <property name="toolTip">
         <string>Steps of a random walk.</string>
        </property>
        <property name="text" stdset="0">
         <string>0</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>sorw</string>
        </property>
       </widget>
      </item>
      <item row="103" column="0">
       <widget class="QLabel" name="label_nbrw">
        <property name="text">
         <string>nbrw: </string>
        </property>
       </widget>
      </item>
      <item row="103" column="1">
       <widget class="QSpinBox" name="nbrwQSpinBox">
        <property name="toolTip">
         <string>Number of random walks starting at each pre-segmented pixel.</string>
        </property>
        <property name="text" stdset="0">
         <string>0</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>nbrw</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearPointsButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Clear points in current layer.</string>
       </property>
       <property name="text">
        <string>Clear Points</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteLabelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove label in current layer.</string>
       </property>
       <property name="text">
        <string>Remove Label</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="biomedisaButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the Biomedisa algorithm.</string>
     </property>
     <property name="text">
      <string>Run Biomedisa</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
